core/arch/arm/plat-rockchip/platform_rk3588.c

```diff
@@ -426,6 +432,16 @@ TEE_Result tee_otp_get_hw_unique_key(struct tee_hw_unique_key *hwkey)
 		res = tee_rockchip_persist_huk(hwkey);
 	}
 
+	huk = malloc(sizeof(*huk));
+	if (!huk) {
+		res = TEE_ERROR_OUT_OF_MEMORY;
+		goto out;
+	}
+	memcpy(huk->data, hwkey->data, HW_UNIQUE_KEY_LENGTH);
+
+cached:
+	/* Copy HUK into hwkey->data */
```

Comment:
Nitpicking: Is the inline comment really needed (instruction below seems quite explicit)?

Reply:
I'll drop the comment. There are several other comments in the code that just describe, what the next line is doing, without adding further information. Should I remove such comments?